api_gateway/test_runtime_quality.py

```
@@ -0,0 +1,184 @@
+  "ipw_type": "IPW_V1",
+  "predictions": [{"action_id": "A", "p": 0.9}, {"action_id": "B", "p": 0.6}],
+  "collapse_threshold": 0.61,
+  "probability_policy": {"requires_normalization": true, "epsilon": 0.0001, "on_violation": "normalize"},
+  "evidence": {"interaction_velocity": 0.73},
+  "unexpected": true
+}
+""",
+                encoding="utf-8",
+            )
+            proc = subprocess.run(
+                ["python", "tools/contracts/contract_checker.py", "--strict"],
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)